yb-voyager/src/srcdb/source.go

```diff
@@ -315,6 +336,93 @@ func (source *Source) extrapolateDSNfromSSLParams(DSN string) string {
 	return DSN
 }
 
+func (source *Source) getPgDumpSchemaArgsFromFile() string {
```

Comment:
Try and merge `getPgDumpSchemaArgsFromFile` and `getPgDumpDataArgsFromFile` into a single function, and parameterise the section?